api/src/pcapi/core/offers/repository.py

```diff
@@ -374,6 +377,17 @@ def venue_already_has_validated_offer(offer: Offer) -> bool:
     )
 
 
+def venues_with_no_offer_since_x_days(number_of_days: int) -> Query:
+    return (
+        db.session.query(Venue.id, Venue.bookingEmail)
+        .outerjoin(Offer, Offer.venueId == Venue.id)
+        .filter(
+            Offer.venueId == None,
```

Comment:
(détail) ça marche, mais il y a une forme :

```suggestion
            Offer.venueId.is_(None),
```

... qu'on préfère car cela nous permettrait de réactiver la vérification "singleton-comparison" de pylint (actuellement désactivée).

Reply:
ok merci